docs/CONTRIBUTING.md

```diff
@@ -32,3 +32,75 @@ Coming soon!
 After values are pushed the cluster's drone will do a deployment run. Helmfile diff is used targeting `$CLOUD` & `$CLUSTER` before the final deployment of the artifacts.
 
 If you have any meaningful additions to this, please let us know!
+
+## Schema validation
+
+### Guidelines/best practices
+
+We intend to be very exact in our speech regarding schema validation. Currently we perform meta-schema validation on the `values-schema.yaml` file. By having accurate and consistent descriptions (using the `description` keyword), we can provide the user with a better experience in his configuration management of the `otomi-values` repository.
+
+[Under the Apache V2.0 license agreement we are free to distribute the following content:](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#validation)
+
+API objects are validated upon receipt by the apiserver. Validation errors are flagged and returned to the caller in a Failure status with reason set to Invalid. In order to facilitate consistent error messages, we ask that validation logic adheres to the following guidelines whenever possible (though exceptional cases will exist).
```

Comment:
schema validation is not related to the k8s apiserver, so again I don't understand why this is brought inhere

Reply:
This is just the header for the schema guidelines by the Kubernetes docs. Furthermore, after meta schema validation, they should be validated against the Kubernetes API server. So I thought adding this bit will make people understand the eventual target, don't you agree?